conf/test_no_igg.config

```diff
@@ -32,5 +32,5 @@ params {
     spikein_bowtie2    = 'https://raw.githubusercontent.com/nf-core/test-datasets/cutandrun/reference/genomes/e_coli_U00096_3.tar.gz'
 
     minimum_alignment_q_score = 10
-    igg_control               = false
+    use_control               = false
```

Comment:
why not renaming this file to `test_no_control` to follow the same logic?

Reply:
good point, changed